pkg/object/object.go

```diff
@@ -45,9 +48,34 @@ type OpenCompose struct {
 	Volumes  []Volume
 }
 
+func validateReplicas(replicas *int32) error {
+	if replicas != nil && *replicas < 0 {
+		return fmt.Errorf("Replica count is negative")
+	}
+	return nil
+}
+
+func (s *Service) Validate() error {
+	// validate service name, like it cannot have underscores, etc.
+
+	// validate containers
+
+	// validate replicas
+	if err := validateReplicas(s.Replicas); err != nil {
```

Comment:
Stupid question as usual 😄 
If replica check is just one line, and it is used in one place, does it make sense to have a separate function for it? Wouldn't be code more readable if actual check was here? 😇